ru/mail/chrome/messenger/messengercompose/addressingWidgetOverlay.dtd

```diff
@@ -1,4 +1,4 @@
-<!-- address labels -->
+﻿<!-- address labels -->
```

Comment:
I have changed the character encoding for this file from ANSI to UTF-8 (I think that all RU files should be UTF-8).